docs/external-tor-usage.md

```diff
@@ -0,0 +1,49 @@
+# Using External Tor with Haveno
+>Looking to setup external Tor for a seednode? Go [here](https://github.com/haveno-dex/haveno/blob/master/docs/deployment-guide.md#install-tor) for tailored instructions.
+## Setting up Tor:
+
+### 1. Download and Install Tor
+#### Linux
+You can install Tor through your system's package manager, or directly by going to torproject.org. Note that your package manager's version of tor might be outdated or non-GNU (see below note).
+#### MacOS
+You can install Tor through Homebrew (brew install tor), or directly by going to torproject.org. Note that the Homebrew version of tor might be outdated or non-GNU (see note below).
```

Comment:
Change to \`brew install tor\` with the backticks, so it's clear it's a command.